g2core/stepper.cpp

```diff
@@ -1042,6 +1042,31 @@ stat_t st_get_pwr(nvObj_t *nv)
 	return (STAT_OK);
 }
 
+stat_t st_set_sp(nvObj_t *nv)            // set motor step polarity
+{
+    if (nv->value < 0) { return (STAT_INPUT_LESS_THAN_MIN_VALUE); }
+    if (nv->value > 1) { return (STAT_INPUT_EXCEEDS_MAX_VALUE); }
+
+    uint8_t motor = _get_motor(nv->index);
+    if (motor > MOTORS) { return STAT_INPUT_VALUE_RANGE_ERROR; };
+
+    Motors[motor]->setStepPolarity((ioMode)nv->value);
+    return (STAT_OK);
+}
+
+stat_t st_get_sp(nvObj_t *nv)            // get motor step polarity
+{
+    if (nv->value < 0) { return (STAT_INPUT_LESS_THAN_MIN_VALUE); }
+    if (nv->value > 1) { return (STAT_INPUT_EXCEEDS_MAX_VALUE); }
+
```

Comment:
Seems like this would allow values like .5, with I know not what consequences.

Reply:
Hmm, good point. @aldenhart do you feel we should do more here?